.travis.yml

```diff
@@ -15,5 +15,5 @@ before_install:
   - tools/travis/nuke_pg.sh
 install:
   - tools/travis/install_pg.sh
-script: tools/travis/pg_travis_multi_test.sh
+script: CFLAGS=-Werror tools/travis/pg_travis_multi_test.sh
```

Comment:
Would it make sense to put this into the script or are there cases where we don't want -Werror? It looks a bit out-of-place here.